v2/Dockerfile

```diff
@@ -1,4 +1,13 @@
-FROM quay.io/cybozu/ubuntu:20.04
+FROM --platform=$BUILDPLATFORM golang:1.18-buster as build-env
```

Comment:
Can we use `golang` image from Docker Hub?

Reply:
Sorry, I would like to confirm, but does that mean that I should change it to use the `golang` image instead of the `golang:1.18-buster` image?

Reply:
Not really. I mean we probably should use `quay.io/cybozu/golang` as it is now a multi-platform image.

Reply:
When you started this PR, we were not able to use `quay.io/cybozu/golang` for a multi-platform.
But now we can use it for a multi-platform.
Could you change the image? 

https://quay.io/repository/cybozu/golang?tab=tags